autotask/pricers-bot/defender-config.js

```diff
@@ -0,0 +1,109 @@
+const fs = require('fs')
+require('dotenv').config({ path: '././.env' })
+var enums = require('./enums');
+
+
+const infuraKey = fs.existsSync('./.infuraKey')
+  ? fs
+      .readFileSync('.infuraKey')
+      .toString()
+      .trim()
+  : "put infura key inside a .infuraKey file, don't paste it here" // infura key
+
+
+module.exports = {
+
+  gammaNetworkDefaults : {
+          mainnet:  {
+              oracleAddress: '0x789cD7AB3742e23Ce0952F6Bc3Eb3A73A0E08833',
+              addressBookAddress: '0x1E31F2DCBad4dc572004Eae6355fB18F9615cBe4',
+              whitelistAddress: '0xa5EA18ac6865f315ff5dD9f1a7fb1d41A30a6779',
+              strikeAddress: '0xa0b86991c6218b36c1d19d4a2e9eb0ce3606eb48',
+              baseAssetDefaultBotAddress: '0xfacb407914655562d6619b0048a612b1795df783',
+              blockchainExplorerApiUrl: `https://api.etherscan.io/api?`,
+              blockchainExplorerApiKey: process.env.ETHERSCAN_API,
+              nodeUrl:`https://mainnet.infura.io/v3/${infuraKey}`
+          },
+          kovan:  {
+            oracleAddress: '0x32724C61e948892A906f5EB8892B1E7e6583ba1f',
+            addressBookAddress: '0x8812f219f507e8cfe9d2f1e790164714c5e06a73',
+            whitelistAddress: '0x9164eb40a1b59512f1803ab4c2d1de4b89627a93',
+            strikeAddress: '0x7e6edA50d1c833bE936492BF42C1BF376239E9e2',
+            baseAssetDefaultBotAddress: '0x69ba16f3c3c4711321f50ceecb6292985d35025b',
+            blockchainExplorerApiUrl: `https://api-kovan.etherscan.io/api?`,
+            blockchainExplorerApiKey: process.env.ETHERSCAN_API,
+            nodeUrl:`https://kovan.infura.io/v3/${infuraKey}`
+        }
+
+  },
+
+  bots: {
+    1: {
+      name: 'Chainlink pricer bot: Mainnet',
+      chain: 'mainnet',
+      type: enums.BaseAsset,
+      resourcePath: 'chains/mainnet',
+      autoTaskId:'3e380513-1dc3-4e3b-b76e-43faee671bb3',

```

Comment:
is it ok to have these publicly? Will this allow anyone to modify these?

Reply:
Our Defender AUTOTASK_API_KEY and AUTOTASK_API_SECRET will be needed to make changes via this script, Hence i think its safe to have the ids here